en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-graphql-api-in-choreo-connect.md

```diff
@@ -0,0 +1,121 @@
+
+4. Open **APIs** from the top menu and select your API.
+
+5. Open the **Subscriptions** tab from the left menu bar, click on **PROD KEYS**, and generate keys.
+
+6. Open the **Try Out** tab and click **GET TEST KEY**. This will include the access token in the cURL command you generate in the section below.
+
+!!! tip 
+    To generate a temporary test key to invoke the API, follow the steps [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/security/generate-a-test-jwt/)
+
+### Step 4 - Invoke the API by providing a valid GraphQL query
+
+!!! note
+    Currently Choreo Connect supports `query` and `mutation` GraphQL operations only. Upcoming releases will support `subscription` operation also.
+
+1. Since GraphQL APIs require a GraphQL query to provide results, you need to have a valid query. A sample query is provided in below section. In API Manager publisher portal, GraphiQL is used for build queries. You can use explorer option available there to build queries. More information regarding valid GraphQL query generation includes in [here]({{base_path}}/tutorials/create-and-publish-a-graphql-api/#step-51-optionally-try-out-a-query-operation).
+
+``` 
+query{
+   human(id:1000){
+      id
+      name
+   }
+   droid(id:2000){
+      name
+      friends{
+          name
+          appearsIn
+      }
+   }
+}
+```
+2. Click **Execute**.
+
+You will get a response for the GraphQL API invocation, as shown in below.
+
+[![Response of GraphQL Query]({{base_path}}/assets/img/learn/graphql-response-query.png)]({{base_path}}/assets/img/learn/graphql-response-query.png)
+
+If you inspect the network calls from your browser developer tools, you can see the messages passed between the GraphiQL client and the backend.
+    
+
+
+## Via apictl for Standalone Mode
+
+The CLI tool ([**apictl**]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-apictl)) does not support initializing projects for Streaming APIs yet. However, you can download a GraphQL API from a WSO2 API Manager instance and deploy the downloaded project in Choreo Connect. You do not need any special configurations in API-M or Choreo Connect for this. If you do not have a running API Manager instance already, simply download the pack from the [official site](https://wso2.com/api-manager/) and start one locally.
+
+The following are the steps to download and deploy the API using apictl.
+
+### Step 1 - Create an API 
+
+Create a GraphQL API following the steps in [Create a GraphQL API in API Manager](#step-1-create-a-graphql-api-in-api-manager). 
+
+### Step 2 - Download the API
+
+After creating the API and making the necessary changes, click **Download API** in the top right corner. The API will be downloaded as a .zip file and after that it can be deployed via apictl using the next few commands.
+
+{! ./includes/deploy/cc-deploy-api-standalone-mode.md !}
```

Comment:
Explain the requirement to add CC with APICTL.
https://github.com/wso2/docs-apim/pull/6072/files#diff-3824711a0495de826c9c78e6712b7f58bc5c7a4a7d6148e21b8387d8c8d06dd8R10

Reply:
remove include.